---
layout: page
title: Faryabi Lab @ UPENN 
permalink: /index.html
---

![enhancer in cancer](assets/MYC_FISH_EP.jpg){:height="200%" width="200%"} 

Our lab focuses on understanding the mechanisms of epigenetic desregulation in cancer. We are particularly interested in understanding how signaling-dependent transcription factors such as Notch reorganize cancer epigenomes. This mechanistic knowledge constitutes an important step toward leveraging selective disruption of epigenetic homeostasis as targeted therapeutic options. 

We have setup various genomic assays to understand how oncogenic transcription factors regulate linear and 3D epigenome of cancer cells. To this end, we use data-rich assays (ChIP-seq, ATAC-seq, Cut&Run, etc.) to measure chromatin activity and combine these information with high resolution chromatin conformation capture technologies (HiChIP, 4C-seq, etc.). 

We also use single-cell measurements to study how chromatin state (scATAC-seq) and folding (3D DNA FISH) heterogeneity impact cancer cell evolution and response to targeted therapies.

<span style="background-color:rgb(230, 230, 255)">Passionate individuals who are interested in investigating mechanisms of cancer epigenetic deregulation are invited to join us to work on projects supported by National Cancer Institute, Komen Foundation, and Concern Foundation amoung others.</span>
Rotation students and postodcs could use high resolution genomic approaches to investigate the impact of epigenetic deregulation on oncogenesis and treatment response in Notch-mutated cancers. For instance, they can study 
+ How does oncogenic Notch instruct cancer genome folding?
+ What are the underlying mechanisms of epigenetic dysregulation selectivity in Notch-mutated cancers?
+ How does precise epigenetic targeting can be harnessed for cancer therapeutics?
+ How does epigenetic plasticity drive resistance to targeted therapies in Notch-mutated cancers?

Read about our [research](research.html) and checkout our available [positions](positions.html) if you are interested in joining our team. 

<br>

----
<strong><span style="font-size: 1.5em; font-weight: bold; color: #0000cc; background-color: #ffffff">NEWS:</span><strong>

+ Join me to congratulate Jelena for joining Genentech as a research scientist. You will be missed (November 2019)!

+ Our R01 was scored in the NCI R37 fundable range (July 2019)!

+ Babak presented at the National Cancer Institute's Genome Integrity Seminar Series (June 2019).

+ Congratulations to [Yeqiao Zhou](https://twitter.com/YeqiaoZhou) for passed her prelim with the highest score possible (May 2019).

+ Thank's Rutgers Universityâ€™s Molecular and Biochemistry Society Seminar Series for hosting Babak (April 2019).

+ Babak presented at the [Keystone 3D Genome](https://goo.gl/WJSQoX) meeting, March 2019, at Banff.

+ Congratulations to Jelena and Yeqiao! Their Molecular Cell paper entitled ''[Oncogenic Notch Promotes Long-Range Regulatory Interactions within Hyperconnected 3D Cliques](https://goo.gl/WwfaHD)'' is now online.  

+ The significance of our work entitled ''[Classes of ITD predict outcomes in AML patients treated with FLT3 inhibitors](http://clincancerres.aacrjournals.org/cgi/content/abstract/1078-0432.CCR-18-0655)'' was highlighted in a Clinical Cancer Research commentary entitled ''[Prognostic Models Turn the Heat(IT)up on FLT3-ITD-Mutated AML](http://clincancerres.aacrjournals.org/content/25/2/460?iss=2)''. 

+ Gregory is presenting [TooManyCells](https://github.com/faryabiLab/too-many-cells) at Single Cell Biology Keystone Conference (January 2019). 

+ Our preprint describing [TooManyCells](https://github.com/faryabiLab/too-many-cells), a suite of graph-based scRNA-seq clustering and visualization tools, are now on [Biorvix](http://biorxiv.org/cgi/content/short/519660v1) (January 2019). 

+ Acceptance letter from Molecular Cell was Santa's present for Jelena and Yeqaio. Have a wonderful 2019! 

+ Our work entitled ''[Classes of ITD predict outcomes in AML patients treated with FLT3 inhibitors](http://clincancerres.aacrjournals.org/cgi/content/abstract/1078-0432.CCR-18-0655)'', has been accepted for publication in Clinical Cancer Research. A related video clip at the [Penn Center for Precision Medicine](https://www.youtube.com/watch?v=vqLXfHXP98A&index=20&list=PLRGEsZTjjo8mLReT4vqHU6cc_d4hn46GV) (July 2018).

+ Congratulations to Gregory and the rest of the team for their interesting work, ''[Differential integration of transcriptome and proteome identifies pan-cancer prognostic biomarkers](https://goo.gl/pvZ4Dn)'', published at the Frontiers in Genetics. (June 2018)

+ A preprint of the Gregory's manuscript titled ''Classes of ITD predict outcomes in patients with AML treated with FLT3 inhibitors'' is now on [bioRxiv](https://www.biorxiv.org/content/early/2018/05/14/322354). (June 2018)

+ The lab received the Susan G. Komen's New Treatments for Drug-Resistant Breast Cancers award to study the mechanisms of epigenetic dysregulation in breast cancer. (April 2018). 

+ Gregory recieved travel award to present his work at the Biology of Genomes Cold Spring Harbor Conference (April 2018).  

+ Yeqiao joined the lab as a PhD student. Welcome Yeqiao! (March 2018)

+ A short video clip summarizing our AML precision prognostic project is posted at the [Penn Center for Precision Medicine](https://www.youtube.com/watch?v=vqLXfHXP98A&index=20&list=PLRGEsZTjjo8mLReT4vqHU6cc_d4hn46GV) (January 2018).

+ Gregory's software was highlighted at the [Penn Medicine News](https://www.pennmedicine.org/news/news-releases/2017/july/penn-center-for-precision-medicine-awards-565-000-in-second-accelerator-grants)

+ [Abramson Family Cancer Research Institute](http://www.afcri.upenn.edu/) funded our project to investigate the mechanism of epigenetic dysregulation in breast cancer. (November 2017)

+ Congratulations to Jelena and Yeqiao for their great work, ''A unique B-cell regulome links Notch to downstream oncogenic pathways in small B-cell lymphoma'', that was just accepted to Cell Reports. (September 2017) 

+ Welcoming new member of the lab: [10X Genomics Chromium](https://www.10xgenomics.com/single-cell/) (August 2017)

+ Thanks to Jelena Petrovic and Yeqiao Zhou, our first map of breast cancer genome topology is ready! (June 2017) 

+ Our lab is featured on the [Pathology News](http://pathology.med.upenn.edu/news/faryabi-lab-receives-grant-transdisciplinary-awards-program-translational-medicine-and). 

+ Our paper titled [''Functional proteogenomics reveals biomarkers and therapeutic targets in lymphomas''](http://www.pnas.org/content/114/25/6581.short) is now online at PNAS.

+ Penn Center for Precision Medicine [(PCPM)](https://lp.pennmedicine.org/precision-medicine/general?hcmacid=pm) funded our proposal to study precision clinical decision making in targeted therapy of AML.

+ Thomas Campbell started his rotaion at the first week of May 2017.

+ Ben Kahn and Rohan Alur joined the lab for their independent studies in February 2017. Welcome Ben and Rohan.

+ The lab received the Cooper Scholar Award from the [Abramson Cancer Center's](https://www.pennmedicine.org/cancer). 

+ Our Penn Epigenetics Pilot Grant was renewed till 2018. Thanks [The PENN Epigenetics Institute](https://hosting.med.upenn.edu/epigenetics/)!

+ Institute for Translational Medicine and Therapeutics [(ITMAT)](http://www.itmat.upenn.edu/) supported our proposal for single cell epigenomics study of Leukemia.

+ After more than a year researching in the lab, Yeqiao Zhao was admitted to the Genetics & Epigenetics PhD program at UPenn. Congratulations Yeqiao! 


<br> 


