---
layout: page
title: Positions
permalink: /positions.html
---

We are inviting experimental and computational biologists to join our enthusiastic and goal-oriented team to study mechanisms of epigenetic deregulation and genome misfolding in cancer. 

We use combination of cutting-edge chromatin conformation capture (HiChIP, HiC, 4C-seq, etc.), high-content Oligopaint 3D DNA FISH, and single-cell epigenomics to systematically tackle fun and fundamental questions in breast, T and B cell cancers: 

1. How control of gene expression is disrupted

2. Why transcriptional addiction is developed 

3. How heterogeneity and plasticity of transcriptional dependencies enable drug resistance

* **Experimental graduate student and postdocs** will use population and/or single-cell assays to elucidate how oncogenic transcription factors regulate activity and topology of chromatin in breast and B/T-cell cancers. In addition to genetic tools, we use data-rich and high resolution genomic and imaging approaches to investigate the impact of oncogenic Notch on chromatin activity and folding.

* **Computational graduate students and postdocs** will develop algorithms for integration and interrogation of epigenetic and chromatin conformation data sets generated in our lab. The goal of these studies is to precisely identify how oncogenic Notch and its partners control cancer epigenome. In addition to sequencing data, our lab uses high-content imaging for 3D genome topology mapping. Trainees interested in the development of transfer learning algorithms for 3D DNA FISH analysis are strongly encouraged to contact us.

Our lab is part of the, [Penn Epigenetics Institute](https://hosting.med.upenn.edu/epigenetics/program-members/), [Abramson Family Cancer Research Institute](http://www.afcri.upenn.edu/), and [Penn Institute of Biomedical Informatics](http://upibi.org/). These organizations provide ample opportunities for collaboration and scientific exchange with a large community of translational and basic scientists.   

Applicants are encouraged to contact Dr. Faryabi directly or other lab members to learn about details of existing projects and opportunities.
