---
layout: page
title: Research
permalink: /research.html
---

**What is our objective?** Despite the recent advances in cataloging frequent pathogenic mutational events in cancer, the efforts of scientific community to design effective and personalized treatment options are often hindered by the complexity of the disease. This is stemmed partly from our limited understanding of complex and tumor-specific epigenetic circuitry that interact with the genetic events. We exploit systems biology approaches to map epigenetic changes in tumors due to oncogenic signals with the goal of elucidating effects of pathogenic events beyond the genomic mutations. Our goal is to uncover tumor-specific regulatory programs driving tumorigenesis.

**What is our approach?** High-throughput measurement technologies have made it possible to collect systems-level genomics, epigenomics, and chromatin conformation data sets. These high-dimensional measurements revealed unprecedented complexity and tumor specificity of epigenetic dysregulation. Our lab uses both population-based assays such as HiChIP, ChIP-seq, ATAC-seq, and RNA-seq as well as single-cell (sc)ATAC-seq and RNA-seq to elucidate dysregulation in regulome and connectome caused by mutations in cell receptors signaling. Our computational expertise enables us to also benefit from large data already generated by cancer and developmental biologists and deposited in public domains. We harness the trove of public data to improve our hypotheses and strengthen our observations. 

The projects in the lab are broadly categorized into the following topics: 

----

**Understanding Notch-driven epigenetic dysregulation in cancer**

![Epegenetics in Cancer](assets/epigenetics.png){:height="290px" width="300px"} 

Our lab deploys data-rich experimental techniques in combination with novel computational methods to model transcriptional dysregulation in Notch-mutated tumors. To create integrative maps of tumor-specific epigenetic regulatory progam driving gene expression, we measure epigenetic state and 3D chromatin conformation of Notch-mutated tumors and integrate these multi-dementional data sets using concepts from data analytics, mathematical modeling, and machine learning. By interrogation of these multi-omics models, we elucidate the role of epigenetic and chromatin conformation dysregulation in tumor development and adaptation. We leverage this knowledge to harness epigenetic vulnerabilities as effect treatments in Notch-driven malignancies.

----

**Mechanisms of drug resistant in cancer using single-cell epigenomics**

![Singel Cell Epegenetics in Cancer](assets/singleCell.jpg){:height="290px" width="300px"} 

To advance the development of novel multi-therapeutic strategies for acute lymphoblastic leukemia, we use single-cell transposase-accessible chromatin sequencing (scATAC-seq) to understand the mechanism of resistance due to the heterogeneity in tumors cells regulatory regions. To fully benefit from our scATAC-seq data, we also develop novel computational approaches with analysis of this data. 

----

**Medical decision making algorithm in AML**

![FLT3 in AML](assets/precision.png){:height="290px" width="300px"} 

Faryabi lab is also interested in developing computational oncology frameworks to enrich clinical significance of diagnostic tumor genomics to advance the paradigm of personalized medicine. To this end, we leverage clinical cases to investigate the correlation between the heterogeneity in mutational structures and response to targeted therapies. Adopting a “bedside to bench and back” approach, our aim is to identify tumor cell-specific vulnerabilities that could be exploited therapeutically.

