---
layout: page
title: Research
permalink: /research.html
---

**What is our objective?** Despite the recent advances in cataloguing frequent pathogenic mutational events in hematologic malignancies, the efforts of scientific community to design effective and personalized treatment options are often hindered by the complexity of the disease. This could be stemmed from our limited understanding of complex and tumor-specific epigenetic circuitry that interact with the genetic events. We exploit systems biology approaches to map epigenetic changes in tumors due to oncogenic signals to better understand the effect of the pathogenic events beyond the genomic mutations. Our goal is to uncover tumor-specific regulatory programs associated with the pathobiology of the hematologic malignancies and their response to therapeutic interventions.

**What is our approach?** High-throughput measurement technologies have made it possible to collect systems-level genomics, epigenomics, and functional genomics data on various tumor types. These high-dimensional measurements revealed unprecedented complexity and tumor specificity of epigenetic dysregulation. Our lab uses both population-based assays such as DNA-seq, ChIP-seq, ATAC-seq, and RNA-seq as well as single-cell (sc)ATAC-seq. Our computational expertise enables us to also benefit from large data already generated by cancer and developmental biologists and deposited in public domains. We harnest the trove of public data to improve our hypotheses and strengthen our observations. 

The projects in the lab are broadly categorized into the following topics: 

----

**Understanding epigenetic dysregulation in cancer in response to aberrant Notch signaling**

![Epegenetics in Cancer](assets/epigenetics.png){:height="290px" width="300px"} 

To create integrative map of tumor-specific epigenetic programs associated with particular oncogenic signals, our lab develops computational methods to relate disparate "omics" data sets by borrowing concepts from data analytics, mathematical modeling, and machine learning. By interrogation of these multi-omics models, we elucidate the role of epigenetic dysregulation in tumor development and adaptation and leverage this knowledge to harness epigenetic vulnerabilities as effect therapeutic options in hematologic malignancies.

----

**Mechanisms of drug resistant in cancer using single-cell epigenomics**

![Singel Cell Epegenetics in Cancer](assets/singleCell.jpg){:height="290px" width="300px"} 

To advance the development of novel multi-therapeutic strategies for acute lymphoblastic leukemia, we use single-cell transposase-accessible chromatin sequencing (scATAC-seq) to understand the mechanism of resistance due to the heterogeneity in tumors cells regulatory regions. To fully benefit from our scATAC-seq data, we also develop novel computational approaches with analysis of this data. 

----

**Medical decision making algorithm in AML**

![FLT3 in AML](assets/precision.png){:height="290px" width="300px"} 

Faryabi lab is also interested in developing computational oncology frameworks to enrich clinical significance of diagnostic tumor genomics to advance the paradigm of personalized medicine. To this end, we leverage clinical cases to investigate the correlation between the heterogeneity in mutational structures and response to targeted therapies. Adopting a “bedside to bench and back” approach, our aim is to identify tumor cell-specific vulnerabilities that could be exploited therapeutically.
